DBZ-5290 Upgrade SQL Server driver to 10.2.1.jre8 (#3639)
debezium/debezium: documentation/modules/ROOT/pages/connectors/sqlserver.adoc

For {prodname} to capture change events from SQL Server tables, a SQL Server administrator with the necessary privileges must first run a query to enable CDC on the database.
The administrator must then enable CDC for each table that you want Debezium to capture.

[NOTE]
====

> The Microsoft JDBC SQL server uses SSL encrypted connections by default.

Reviewer: ```suggestion
By default, JDBC connections to Microsoft SQL Server are protected by SSL encryption.
```

> If the SQL Server database does not have SSL enabled, or you wish to connect without using SSL, please specify the `database.encrypt=false` option in the connector configuration to disable SSL support.

Reviewer: ```suggestion
If SSL is not enabled for a SQL Server database, or if you want to connect to the database without using SSL, you can disable SSL by setting the value of  the `database.encrypt` property in connector configuration to `false`.
```

|11
|The name of the database history topic where the connector will write and recover DDL statements. This topic is for internal use only and should not be used by consumers.

|12

> |The SSL truststore where the server certificates are stored. This is required when `database.encrypt=false` is not specified.

Reviewer: ```suggestion
|The path to the SSL truststore that stores the server's signer certificates. 
This property is required unless database encryption is disabled (`database.encrypt=false`).
```

|13

> |The SSL truststore password. This is required when `database.encrypt=false` is not specified.

Reviewer: ```suggestion
|The SSL truststore password. 
This property is required unless database encryption is disabled (`database.encrypt=false`).
```

<9> A list of all tables whose changes {prodname} should capture.
<10> The list of Kafka brokers that this connector will use to write and recover DDL statements to the database history topic.
<11> The name of the database history topic where the connector will write and recover DDL statements. This topic is for internal use only and should not be used by consumers.

> <12> The SSL truststore where the server certificates are stored. This is required when `database.encrypt=false` is not specified.
> <13> The SSL truststore password. This is required when `database.encrypt=false` is not specified.

Reviewer: ```suggestion
<12> The path to the SSL truststore that stores the server's signer certificates. 
This property is required unless database encryption is disabled (`database.encrypt=false`).
<13> The SSL truststore password. 
This property is required unless database encryption is disabled (`database.encrypt=false`).
```
